routers/Pets.js

```diff
@@ -1,30 +1,56 @@
 import { Router } from 'express';
 
+import { escapeId } from 'mysql2';
+
 import Database from '../Database.js';
 
 const connection = Database();
 
-const pet = 'pet_id, user_id, pet_name, type, breed, weight, microchip_id, fur_color'; // listing of columns in pets schema
-
 const router = Router();
 
-router.get('/', (req, res) => {
-  const {
-    mode, offset, page, limit,
-  } = req.query;
-  let sqlOffset = (Number(offset) || 0);
+export const prepareQuery = (fields, limit, page, filterParams) => {
+  let sqlOffset = 0;
   sqlOffset += ((Number(page) || 1) - 1) * (Number(limit) || 20);
 
-  const columns = {
-    pet_details: pet,
-  }[mode] || '*';
+  //console.log(fields);
+  let sqlFields = escapeId(fields);
+  sqlFields = sqlFields.slice(1, -1);
```

Comment:
What's the purpose of this slice?

Reply:
It is to remove the backticks. I just committed a fix that resolves the problem